gems/sorbet/lib/require_everything.rb

```diff
@@ -106,6 +104,29 @@ def self.patch_kernel
 
   private
 
+  def self.rb_file_paths
+    sorbet_exec = File.realpath("#{__dir__}/../bin/srb")
+    output = `#{sorbet_exec} tc -p file-table-json --stop-after=parser --silence-dev-message --no-error-count`
```

Comment:
Let's not use backticks here. This would be the first instance of using backticks for spawning processes in all of `srb init` and I'd not like someone to cargo cult this pattern. Can we stick to `IO.popen` that we were using in all the other places?

More information here:

https://medium.com/zendesk-engineering/running-a-child-process-in-ruby-properly-febd0a2b6ec8

Reply:
Heads up: I pushed this change to your branch.